internal/aws/scripts/PERMISSION_POLICY.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "*",
+            "Resource": "*"
```

Comment:
yes, good point @darrell-d. That is right, it is for testing. We also did make an outright decision to first have it entirely open, since we will need to create and manage infrastructure in the external account, and we didn't want permissions to be a constant blocker, but rather to streamline them once all the required functionality is in place: creating compute nodes, deploying applications etc.